File: PYTHON/1012.py
def triangulo_retangulo(a,c):
    return (a*c) / 2

def circulo(c):
    return c * c * 3.14159

def trapezio(a,b,c):
    area = (a + b)*c
    area /= 2
    return area

def quadrado(b):
    return b ** 2

def retangulo(a,b):
    return a * b

a,b,c = map(float, input().split())

print ("TRIANGULO: %.3f" % triangulo_retangulo(a,c))
print ("CIRCULO: %.3f" % circulo(c))
print ("TRAPEZIO: %.3f" % trapezio(a,b,c))
print ("QUADRADO: %.3f" % quadrado(b))
print ("RETANGULO: %.3f" % retangulo(a,b))

File: GO/1012.go
package main

import (
	"fmt"
	"math"
)

const PI float64 = 3.14159

type Data struct {
	A float64
	B float64
	C float64
}

func (d *Data) RectangledTriangle() float64 {
	return (d.A * d.C) / 2.0
}

func (d *Data) Circle() float64 {
	return PI * math.Pow(d.C, 2.0)
}

func (d *Data) Trapezium() float64 {
	return ((d.A + d.B) * d.C) / 2.0
}

func (d *Data) Square() float64 {
	return math.Pow(d.B, 2)
}

func (d *Data) Rectangled() float64 {
	return d.A * d.B
}

func main() {
	var a, b, c float64
	fmt.Scan(&a)
	fmt.Scan(&b)
	fmt.Scan(&c)

	data := Data{a, b, c}

	fmt.Printf("TRIANGULO: %.3f\n", data.RectangledTriangle())
	fmt.Printf("CIRCULO: %.3f\n", data.Circle())
	fmt.Printf("TRAPEZIO: %.3f\n", data.Trapezium())
	fmt.Printf("QUADRADO: %.3f\n", data.Square())
	fmt.Printf("RETANGULO: %.3f\n", data.Rectangled())
}
